File: design/lua/gtk.lua
local lgi = require 'lgi'
local Gtk = lgi.require('Gtk')

print "Hello GObject"

local window = Gtk.Window(bt_get_window())
local box = Gtk.VBox { spacing = 12, border_width = 24 }
box:add(Gtk.Label { label = '---- Hello ----' })
box:add(Gtk.Label { label = '<big><b>Lua</b></big>', use_markup = true })
window:add(box)

File: design/seed/gtk.js
Gtk = imports.gi.Gtk;

print('Hello GObject');

var window = bt_get_window();
var box = new Gtk.VBox({spacing: 12, border_width: 24});
box.add(new Gtk.Label({label: '---- Hello ----'}))
box.add(new Gtk.Label({label:'<big><b>Seed</b></big>', use_markup: true }))
window.add(box);
